configs/linux/Agent/Ubuntu/Ubuntu.Dockerfile

```diff
@@ -58,7 +58,6 @@ RUN apt-get update && \
     curl -SL "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && chmod +x /usr/local/bin/docker-compose && \
     \
     apt-get install -y --no-install-recommends \
-            fontconfig \
```

Comment:
`fontconfig` was installed in `configs/linux/MinimalAgent/Ubuntu/Ubuntu.Dockerfile`, L#22.